core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/StackV4Controller.java

```diff
@@ -1,16 +1,5 @@
 package com.sequenceiq.cloudbreak.controller.v4;
 
-import java.util.ArrayList;
```

Comment:
In this file there is no other change just the order of imports. I would exclude this change from this commit.